core/src/main/java/com/hcl/ecomm/core/servlets/ProductServlet.java

```diff
@@ -33,20 +34,15 @@ protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse r
 
 		
 		try {
-
-			
-			JsonArray responseStream = productService.getAllProductDetails();
-			JsonArray itemsarr= responseStream.getAsJsonArray();
-			LOG.info("itemsarr is {}", itemsarr.toString());
+			JSONArray responseStream = getAllProductDetails();
+			LOG.info("itemsarr is {}", responseStream.toString());
 			List<HashMap<String, String>> list = new ArrayList<HashMap<String, String>>();
-			JsonObject productObject = itemsarr.get(0).getAsJsonObject();
-			LOG.info("productobject is {}", productObject);
-			for(int i=0; i<itemsarr.size();i++){
+			for(int i=0; i<responseStream.length();i++){
```

Comment:
@Subrat-Dash-416  Why those changes required?

Reply:
@rakeshkumar-verma  bcoz we have changed from google json to normal json , so methods changes were required to match the same functionality .